katana-runner.dockerfile

```
@@ -0,0 +1,33 @@
+FROM rust:1.82.0-slim-bookworm 
+
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)